chat-monitor.js

```diff
@@ -49,16 +53,30 @@ var configFields = {
         "type" : "checkbox",
         "default" : true
     },
+    "UsernameColors": {
+        "label" : "Colorful usernames",
+        "type" : "checkbox",
+        "default" : false
+    },
+    "AlignUsernames": {
+        "label" : "Align usernames",
+        "type" : "checkbox",
+        "default" : false
+    },
+    "Badges": {
+        "label" : "Show badges",
+        "type" : "checkbox",
+        "default" : false
+    },
     "UsernamesToHighlight": {
         "label" : "Username(s)",
         "section" : ["Username Highlighting", "Change the color of any of the usernames listed below. Separate with commas"],
         "type" : "text",
         "title" : "seperate usernames with commas, case sensitive",
-        //These as defaults as I can't imagine Paul will mind, and I know I don't mind.
-        "default" : "ThingsOnMyStream, cheetoJack"
+        "default" : ""
```

Comment:
I left these two names as defaults for highlighting as examples for anyone trying to figure it out. The example is better for usability than an empty box.